e2e/wdio/core/tests/select.e2e-spec.ts

```diff
@@ -234,9 +204,6 @@ describe('Select component:', () => {
         });
 
         it('should be able to control select by buttons', () => {
-            if (browserIsFirefox()) {
```

Comment:
for all of these, you made sure it works properly on FF for mac and windows, right?

Reply:
sure, i have run in FF all tests and it passed